pkgs/os-specific/linux/broadcom-sta/default.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation {
     (fetchpatch {
       name = "linux-4.8.patch";
       url = "https://aur.archlinux.org/cgit/aur.git/plain/004-linux48.patch?h=broadcom-wl-dkms";
```

Comment:
I would suggest to choose a stable url like: https://git.archlinux.org/svntogit/community.git/plain/trunk/004-linux48.patch?h=packages/broadcom-wl-dkms&id=8c6d80fc77eb20cbb4fd0eca71fa8a03b9447c3f (points to a commit).